test/common/verifier.go

```diff
@@ -23,6 +23,29 @@ import (
 
 const fileForAuditEvent = "/etc/newfile"
 
+// ExpectedAuditRules are the expected audit rules. Exposed so that they can be modified by tests.
+var ExpectedAuditRules = `## This file is automatically generated from /etc/audit/rules.d
```

Comment:
Wouldn't it be better if expected audit rules are somehow configured in the verifier (passed outside, from the caller) instead of overwriting an exported var?

Reply:
Yeah, I was considering that, since they only differ in one test case I left a "default" variable here so that I don't have to pass them everywhere. But I could change that to keep it in line with other verifier options.